android/src/com/mapzen/tangram/Tangram.java

```diff
@@ -130,11 +136,16 @@ public boolean onDoubleTap(MotionEvent event) {
     }
 
     public boolean onScroll(MotionEvent e1, MotionEvent e2, float distanceX, float distanceY) {
-        
-        // We flip the signs of distanceX and distanceY because onScroll provides the distances
-        // by which the view being scrolled should move, while handlePanGesture expects the 
-        // distances by which the touch point has moved on the screen (these are opposite)
-        handlePanGesture(-distanceX, -distanceY);
+        // Only pan for scrolling events with just one pointer; otherwise vertical scrolling will
+        // cause a simultaneous shove gesture
+        if (e1.getPointerCount() == 1 && e2.getPointerCount() == 1) {
+            // We flip the signs of distanceX and distanceY because onScroll provides the distances
```

Comment:
Dont think this comment is applicable now considering both start and end positions are used.